packages/use-sync-external-store/src/__tests__/useSyncExternalStoreShared-test.js

```diff
@@ -654,8 +654,8 @@ describe('Shared useSyncExternalStore behavior (shim and built-in)', () => {
       {
         withoutStack: gate(flags => {
           if (flags.enableUseSyncExternalStoreShim) {
-            // Stacks don't work when mixing the source and the npm package.
-            return flags.source ? 1 : 0;
+            // `console.error` in this package should not be transformed, so we'll never see stack traces
+            return true;
```

Comment:
The warning already had a test here, so I believe it's not necessary to add one more.

I'm a bit stumped that the existing test didn't crash with React 17 and new builds, though. I believe it should have been running with the npm artefact, too?

Reply:
Probably because it used the UMD build. I'll check why it didn't catch this.

Reply:
Also fails with UMD. We just had the tests for the shim+17 disabled when we enabled "ref as a prop" everywhere: https://github.com/facebook/react/pull/29874